etc/kayobe/kolla/config/octavia/cakey.pem

```diff
@@ -0,0 +1,30 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
not 100% certain how to encrypt this. should this go inside `etc/kayobe/kolla/passwords.yml`?

Reply:
Hopefully you can just use `ansible-vault encrypt`  and it will be automatically decrypted. Probably want a new one now though, right?

Reply:
We could maybe merge the fork, then open a new PR to add new Vault'ed CA key and also the Vault'ed Octavia user SSH private key and pub-key to pre-prod branch and re-deploy Octavia.